### ISSUE TITLE
Course Suggestions

### ISSUE THREAD

**Issue by iampavangandhi** [iampavangandhi/TheNodeCourse#24]
Hello @everyone,

You can tell your thoughts and suggestions related to TheNodeCourse here.

**Pavan Gandhi**
**Mentor**

**Comment by howdyAnkit** [comment]
Hey, @iampavangandhi 

Please check Below the mentioned Automated Bots Below.
it will be the perfect use for our Project.

- [x] Hound (Automatic Code Review)
- [x] ImgBot (Automatic Optimizes Image)
- [x] Codacy (Automated code review)

**Comment by iampavangandhi** [comment]
Hey, @howdyAnkit 

Thanks for the suggestions we don't need these automated bots in TheNodeCourse Project because I and @vectorrb are the only contributors. We will surely use these bots in TradeByte as it's a team project.

**Comment by ChiragBolakani** [comment]
Hey @iampavangandhi I have a course suggestion of having a tutorial of bcrypt in TheNodeCourse as its widely used in hashing the credentials and then using it in storing the data in MongoDB. I would like to work on this as I'm pretty familiar with NodeJs.

**Comment by iampavangandhi** [comment]
@ChiragBolakani Yes, you can try. you also have to create a `.md` file where you explain all about bcrypt and your code in a `code` folder then add both of them in a folder `Auth Topic 6` and add it in the `04 Auth and Database` folder

Note: You don't have to do any linking. I will do that myself after merging the PR

**Comment by ChiragBolakani** [comment]
> @ChiragBolakani Yes, you can try. you also have to create a `.md` file where you explain all about bcrypt and your code in a `code` folder then add both of them in a folder `Auth Topic 6` and add it in the `04 Auth and Database` folder
> 
> Note: You don't have to do any linking. I will do that myself after merging the PR
I'll start working on it

**Comment by ChiragBolakani** [comment]
@iampavangandhi I have started working on it.

**Comment by ChiragBolakani** [comment]
@iampavangandhi I have completed the task. This is my pull request #38

**Comment by iampavangandhi** [comment]
@ChiragBolakani PR Merged

**Comment by ChiragBolakani** [comment]
@iampavangandhi Please add the hacktober label again to the repo. My pull request was not counted as the repo didn't had hacktoberfest label. The hacktoberfest also has a 14 days review period and the pull request needs to pass that. You may remove the hacktoberfest label after 10 November. Thank you.
